modules/cli/src/main/scala/scala/cli/commands/packaging/Spark.scala

```diff
@@ -0,0 +1,22 @@
+package scala.cli.commands.packaging
+
+import dependency._
+
+object Spark {
+
+  private def names = Seq(
+    // FIXME Add more? (see "cs complete-dependency org.apache.spark: | grep '_2\.12$'")
```

Comment:
@prolativ should we include more?

Reply:
Do we need to list all existing spark modules here or only the most commonly used ones?

Reply:
These are the modules we exclude from the assemblies generated by `scala-cli package --spark`. Basically, we want to exclude modules whose JARs are already in the Spark distribution.